examples/breakout-rooms/src/components/BreakoutStarter/BreakoutStarter.css

```
@@ -11,21 +11,46 @@
 }
 
 .starter-action {
-  background: #f2f4fc;
-  border-radius: 0.25em;
-  padding: 1em;
+  background: var(--blue200);
+  border: none;
+  border-radius: var(--border-radius-medium);
+  cursor: default;
   display: flex;
+  padding: 1em;
   gap: 1em;
   align-items: center;
   align-self: flex-start;
   width: 100%;
 }
 
 .starter-action .starter-action-title {
-  font-size: 1em;
+  font-size: 1.25em;
   font-weight: 600;
   line-height: 1.5em;
   text-align: left;
-  color: #3859ff;
+  color: var(--blue700);
   margin: 0;
 }
+
+.starter-action:hover {
+  background: var(--blue300);
+  cursor: pointer;
+}
+
+.starter-action:hover .starter-action-title {
+  color: var(--blue900);
+}
+
+.starter-action:hover .rounded-plus {
+  background-color: var(--blue900);
+}
+
+.rounded-plus {
+  border-radius: var(--border-radius-circle);
+  background-color: var(--blue700);
+  padding: 8px;
+
+  & svg {
```

nesting in css isnt supported well enough
```suggestion
}
.rounded-plus svg {
```

---

+1 already removed these locally because of that reason